docs/guide/evaluator.md

```diff
@@ -15,7 +15,7 @@ the [Pusher](pusher.md) that it is ok to push the model to production.
 
 *   Consumes:
     *   An eval split from
-        [ExampleGen](https://www.tensorflow.org/tfx/guide/examplegen)
+        [ExampleGen](https://www.tensorflow.org/tfx/guide/examplegen) or from [Transform](https://www.tensorflow.org/tfx/guide/transform) 
```

Comment:
I think this one could be more accurate and less wordy if it is
`[Examples](https://www.tensorflow.org/tfx/api_docs/python/tfx/v1/types/standard_artifacts/Examples)`.

Reply:
Agreed